subprojects/common/waiter/src/main/kotlin/com/avito/android/util/Waiter.kt

```diff
@@ -1,9 +1,10 @@
 package com.avito.android.util
 
+import com.avito.android.waiter.repeatFor
 import com.avito.android.waiter.waitFor
 
 fun <T> waitForAssertion(
-    timeoutMilliseconds: Long = DEFAULT_TIMEOUT_SEC,
+    timeoutMilliseconds: Long = DEFAULT_TIMEOUT_MS,
```

Comment:
Are 5 seconds not too much?

Reply:
Thought about it. Don't know how to choose an optimal timeout

Reply:
only by collecting enough data.
Do we have some metrics about it?

Reply:
No, we don't. I'll add it if you help me with the correct series name and Graphana visualization.

Reply:
that will be a lot of data. What do we need it for? If only to determine this value i see it as an overkill